src/lighter/main.py

```diff
@@ -243,6 +254,9 @@ def verify(filenames, targetdir=None):
     
     deploy_parser.add_argument('filenames', metavar='YMLFILE', nargs='+',
                        help='Service files to expand and deploy')
+    
+    deploy_parser.add_argument('-p', '--password-check-fail', dest='failOnPassword', help='Fail on password/key check triggering [default: %(default)s]',
```

Comment:
That makes sense. I suck at naming things.

I don't think we need to wait before merging, as it will default to only warn about these things.